test/CMakeLists.txt

```diff
@@ -78,6 +78,15 @@ add_fmt_test(printf-test)
 add_fmt_test(ranges-test ranges-odr-test.cc)
 add_fmt_test(scan-test)
 add_fmt_test(std-test)
+try_compile(compile_result_unused
+            ${CMAKE_CURRENT_BINARY_DIR}
+            SOURCES ${CMAKE_CURRENT_LIST_DIR}/detect-stdfs.cc
+            CMAKE_FLAGS CMAKE_OSX_ARCHITECTURES=${CMAKE_OSX_ARCHITECTURES}
```

Comment:
What is this for?

Reply:
Compiling ``detect-stdfs.cc`` and read compiler output to get fslib info from error message.

Reply:
I mean specifically the above line that forwards `CMAKE_OSX_ARCHITECTURES`.

Reply:
``CMAKE_OSX_ARCHITECTURES`` affects the compiler but I'm not sure if it affects the standard library as well.
Removed it.